pkg/droplets/droplets_test.go

```diff
@@ -208,6 +208,7 @@ var d = pkg.create({
 	private_networking: true,
 	user_data:    "lolll",
 	monitoring:   true,
+	tags:					["test"],
```

Comment:
them tabs! not to start a war but the rest was space so can you make this spaces too :)